staticjinja/reloader.py

```diff
@@ -12,6 +14,9 @@ class Reloader(object):
     """
     def __init__(self, site):
         self.site = site
+        # The following could be part of the Site.__init__ but it would waste
+        # time if the reloader is not used
+        self.site.dep_graph = DepGraph(site)
```

Comment:
This should just be `self.dep_graph`. It's generally a bad idea to dynamically add attributes to classes like this since it becomes quickly unmaintainable.